src/plugins/data/common/search/expressions/esql.ts

```diff
@@ -323,8 +340,11 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
                     ? {
                         appliedTimeRange: input?.timeRange,
                         params: {},
+                        indexPattern,
```

Comment:
Just curious what is this empty `params: {}` for.

Reply:
that was there before, its the configuration for the field formatter.

Reply:
Field formatter for ES|QL?